Attestation/ek-secret.md

```diff
@@ -0,0 +1,174 @@
+# `Passing a secret to a TPM using only the EK`
+
+This is example code to pass a secret to a system by just knowing the endorsenment key.
+We will be using the current (commit 07a92e9fa75548ea102ce90b3b6182093b3f7a73 or later) master branch of https://github.com/tpm2-software/tpm2-pytss
+
+The terms for the systems are `client`, the system we want to pass the secret to and `server`, the system which has the secret but doesn't have any TPM.
+One assumtion that will be made is that you already have the EK for the remote system on the local system, and trust it.
```

Comment:
s/EK/EKpub/

Reply:
I'll fix, sadly the nameAlg is not included in the certificate